src/cloudflare/internal/global.d.ts

```diff
@@ -0,0 +1,3 @@
+declare namespace Cloudflare {
+  const compatibilityFlags: Record<string, boolean>;
+}
```

Comment:
@penalosa ... it should be possible to generate this directly from the compat flags capnp definitions, both for internal and external use. If you can help guide I can do the work ;-) 

the structure is simple...

```
declare namespace Cloudflare {
  const compatibilityFlags: {
    readonly foo: boolean,
    //...
  };
}
```

Where there is one `readonly foo: boolean` entry for every enable flag in the capnp.

Reply:
Yeah something like that should work. I don't think it's worth blocking on it tho. This is something we can incrementally improve on.

Reply:
Going to mark this resolved so I can merge but will unresolve it after so we don't forget ;-)